modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -6055,7 +6055,7 @@
               <c:simple-property name="connectors" />
             </c:list-property>
             <c:simple-property name="socket-binding" required="true" type="string" readOnly="false" description="The broadcast group socket binding.">
-              <c:option-source target="configuration" expression="*/socket-binding=name:type=SocketBindingGroup"/>
+              <c:option-source target="configuration" expression="*/socket-binding=name:type=SocketBindingGroup" expressionScope="baseResource"/>
```

Comment:
@burmanm I added `expressionScope="baseResource"` to all `c:option-source` elements. All of them seems to expect its own resources anyway. Do you think this is too aggressive?